### DISCUSSION TITLE
In wiki.js without login itself my homepage content is coming.

### DISCUSSION THREAD

**Discussion by ganesh112700** [requarks/wiki#5588] [Help / Questions]
In wiki.js without login itself, my homepage content is coming, I installed the wiki.js in ubuntu18.04 machine, and its working as a service, but the issue is without user login itself users can see my wiki.js content, I need a solution for if users try to access the wiki.js the login page should come first,so kindly give a solution for this.

**my-email-ID: ganeshaw11@gmail.com**

**Comment by ashima0512** [comment]
I had the same 'issue' when first installing and using Wiki.js. As a programmer I supposed designer of Wiki made it the permission of anonymous users. So I navigate to Admin-->Groups-->Page Rules, and fill some public path in the last edit text box and save(UPDATE GROUP on the upper-right corner). It works when I logout or access wiki.js with another browser without login, i.e. I only have access to the public path I set.

**Reply by ganesh112700** [reply]
@ashima0512 
         If the user is login into wiki.js at the time that accesses given page is showing, and after the user clicks the logout button means that time all pages are showing, it's not redirecting to the login page.
